scripts/ci/images-v2/Dockerfile.ci-spack-ubuntu20.04-base

```
@@ -0,0 +1,52 @@
+# vim: ft=dockerfile
+FROM ecpe4s/ubuntu20.04:23.02
+
+COPY packages.yaml /etc/spack/packages.yaml
+COPY specs.yaml /tmp/specs.yaml
+
+RUN . /spack/share/spack/setup-env.sh && \
+    export ADIOS_SPEC_NO_MPI="adios2~mpi" && \
+    export ADIOS_SPEC_OPENMPI="adios2+mpi^openmpi@4.1.4" && \
+    export ADIOS_SPEC_MPICH="adios2+mpi^mpich device=ch3 netmod=tcp" && \
+    spack env create adios2-ci-serial /tmp/specs.yaml && \
+    spack -e adios2-ci-serial add ${ADIOS_SPEC_NO_MPI} && \
+    spack -e adios2-ci-serial concretize && \
+    spack -e adios2-ci-serial install \
+      --include-build-deps \
+      --no-check-signature \
+      --fail-fast \
+      --only dependencies \
+      -j$(grep -c '^processor' /proc/cpuinfo) && \
+    spack env create adios2-ci-mpi /tmp/specs.yaml && \
+    spack -e adios2-ci-mpi add ${ADIOS_SPEC_OPENMPI} && \
```

```suggestion
    spack -e adios2-ci-ompi add ${ADIOS_SPEC_OPENMPI} && \
```

Will renaming this across the PR be a big change?